test/Symmetric/McryptTest.php

```diff
@@ -207,4 +208,31 @@ public function testDecryptWithoutKey()
         $this->setExpectedException('Zend\Crypt\Symmetric\Exception\InvalidArgumentException');
         $this->mcrypt->decrypt($this->plaintext);
     }
+
+    public function testSetOptions()
+    {
+        $options = [
+            'algo'    => 'blowfish',
+            'mode'    =>  MCRYPT_MODE_CFB,
+            'key'     => 'test',
+            'iv'      => '12345678',
+            'padding' => 'nopadding'
+        ];
+        $this->mcrypt->setOptions($options);
+
+        $this->assertEquals($options, $this->mcrypt->getOptions());
```

Comment:
I think you could omit this assertion, since the ones that are important already have assertions on the getters.